config.toml

```diff
@@ -920,6 +919,7 @@
     PeerAuthenticationTimeBetweenSendsWhenErrorInSec = 60   # 1min
     PeerAuthenticationThresholdBetweenSends          = 0.1  # 10%
     HeartbeatTimeBetweenSendsInSec                   = 300  # 5min
+    HeartbeatTimeBetweenSendsDuringBootstrapInSec    = 300  # 5min
```

Comment:
```suggestion
    HeartbeatTimeBetweenSendsDuringBootstrapInSec    = 60   # 1min
```
for a better feedback on the real time trie sync status

Reply:
It will spam the network. Decided with @AdoAdoAdo to be left to 5 minutes

Reply:
Anyway, anyone can manually alter it in prefs.toml :)

Reply:
:+1: